### DISCUSSION TITLE
Is it possible to move the camera up and down in the 3D panel?

### DISCUSSION THREAD

**Discussion by Alessio-Parmeggiani** [foxglove/community#939] [Q&A] [answered]
Hi, I noticed that in the 3D panel it is not possible to move the camera up or down using the mouse or using a keyboard shortcut. 
Indeed it is only possible to rotate the camera or move it sideways or forward/backward.
Am I missing something or this feature does not exist?

**Comment by snosenzo** [comment, marked as answer]
@Alessio-Parmeggiani it is not possible using mouse or keyboard, but if you have the 3D settings panel open you can drag over the `Target Z` field and move your mouse left or right to increase or decrease the Z-value of the center of the orbit controls shown in the screenshot. 
<img width="993" alt="Screenshot 2024-02-22 at 10 38 47 AM" src="https://github.com/foxglove/community/assets/10187776/4e98ac0e-8042-4cc0-9cec-12bc73d460e7">

You can also change the zoom by using mouse "scroll" either by mouse-wheel or gesture on trackpad.
Let us know if either of those work for you.

**Reply by Alessio-Parmeggiani** [reply]
Thanks for answering, I knew about this mode but it's not comfortable when exploring a 3D environment, I noticed that using <kbd>W</kbd><kbd>A</kbd><kbd>S</kbd><kbd>D</kbd> does change the values in the other sliders, so I wondered if there was a shortcut also for the Z slider.
